### ISSUE TITLE
New theme similar to Tusky

### ISSUE THREAD

**Issue by Danie10** [tateisu/SubwayTooter#203]
I'm loving Subway Tooter since discovering it - and even did a video about it at https://www.youtube.com/watch?v=BAT66wJZTgQ.

It is without a doubt the most functionally rich Mastodon app available, but I suppose many get put off by it's two basic themes to choose from. They maybe just look a bit flat. So I did got sample from the colours that Tusky uses and I could easily set these colours in the options as per attached image. But I'd like to suggest that a 3rd theme is offered, maybe just called "blue" that is similar to this. It may make the app more attractive to non-power users.

These are the colour codes I'm using:
```
Column Header background colour #444B5D
Column Content background colour #282C37
Link #FE4E92D6
Button accent colour: Boost #FF0BBCD4
Button accent colour: Favourites #FFFFFB00
Button accent colour: Bookmarks #FF99AEC7
Button accent colour: Follow #FF06D3C4
Button accent colour: Follow Request #FF0EB8D6
Button accent colour: Reaction #FF99AEC7
Footer's colour: Button background #444B5D
Status bar colour #444B5D
```
![Screenshot_20221119-122645_Subway Tooter](https://user-images.githubusercontent.com/1153726/202846355-6f9fb735-b1c0-4063-a931-a50e0562e8d5.jpg)

**Comment by tateisu** [comment]
I will consider your suggestion to add a third theme. Thank you.

Background:.

The default theme is adjusted so that it is easy to see in sunlight and does not bury text and icons when an image is displayed as a column background. This is because I myself display background images in my columns and use this app outdoors. Also, the Dark theme has a completely black background, as requested by users to reduce battery consumption of the AMO LEDs.

Yeah, it's obvious that the default theme of this app is not following the trend. The trend to use low contrast colors does not match the above policy.

But those policies are irrelevant for adding a third theme.

Translated with www.DeepL.com/Translator (free version)

**Comment by Danie10** [comment]
Great thank you. Yes I agree with what you say. It is already very flexible by allowing users to tweak their own colours. But yes if you can add that as a 3rd option it may make a big different to many users.

**Comment by tateisu** [comment]
```
    <!-- =================================================== -->
    <!-- Mastodon theme -->

    <color name="Mastodon_colorAccent">#ccFFFFFF</color>
    <color name="Mastodon_colorBackground">#282C37</color>
    <color name="Mastodon_colorButtonBgCw">#5b5b5b</color>
    <color name="Mastodon_colorButtonSelected">#ff000080</color>
    <color name="Mastodon_colorButtonAccentBoost">#158297</color>
    <color name="Mastodon_colorButtonAccentFavourite">#a8ab16</color>
    <color name="Mastodon_colorButtonAccentBookmark">#9a151a</color>
    <color name="Mastodon_colorButtonAccentFollow">#FF06D3C4</color>
    <color name="Mastodon_colorButtonAccentFollowRequest">#f00</color>
    <color name="Mastodon_colorButtonAccentReaction">#FF99AEC7</color>
    <color name="Mastodon_colorColumnHeaderAcct">#e4e4e4</color>
    <color name="Mastodon_colorColumnHeaderName">#e4e4e4</color>
    <color name="Mastodon_colorColumnHeaderPageNumber">#e4e4e4</color>
    <color name="Mastodon_colorColumnListDeleteBackground">#FF0000</color>
    <color name="Mastodon_colorColumnListDeleteText">#fff</color>
    <color name="Mastodon_colorColumnListDragHandleBackground">#80444444</color>
    <color name="Mastodon_colorColumnListItemText">#66FFFFFF</color>
    <color name="Mastodon_colorColumnSettingBackground">#383838</color>
    <color name="Mastodon_colorColumnStripBackground">#444B5D</color>
    <color name="Mastodon_colorContentText">#dddddd</color>
    <color name="Mastodon_colorImageButton">#ccFFFFFF</color>
    <color name="Mastodon_colorConversationMainTootBg">#2000a2ff</color>
    <color name="Mastodon_colorLink">#FE4E92D6</color>
    <color name="Mastodon_colorPostFormBackground">#222</color>
    <color name="Mastodon_colorPrimary">#333</color>
    <color name="Mastodon_colorPrimaryDark">#444B5D</color> <!-- ステータスバー背景 -->
    <color name="Mastodon_colorProfileBackgroundMask">#C0000000</color>
    <color name="Mastodon_colorRefreshErrorBg">#D222</color>
    <color name="Mastodon_colorRegexFilterError">#f00</color>
    <color name="Mastodon_colorReplyBackground">#333</color>
    <color name="Mastodon_colorRippleEffect">#777</color>
    <color name="Mastodon_colorSearchFormBackground">#333370</color>
    <color name="Mastodon_colorSettingDivider">#66FFFFFF</color><!-- ダイアログ背景が#424242なので、それより明るくないといけない -->
    <color name="Mastodon_colorShowMediaBackground">#222</color>
    <color name="Mastodon_colorShowMediaText">#ccFFFFFF</color>
    <color name="Mastodon_colorStatusButtonsPopupBg">#424242</color>
    <color name="Mastodon_colorThumbnailBackground">#20ffffff</color>
    <color name="Mastodon_colorTimeSmall">#BBBBBB</color>
    <color name="Mastodon_colorVectorDrawable">#e4e4e4</color>
    <color name="Mastodon_color_column_header">#444B5D</color>
    <color name="Mastodon_list_item_bg_pressed_dragged">#AA444444</color>

    <color name="Mastodon_colorSwitchOff">#777</color>

    <!-- スプラッシュ画面の背景色はテーマ関係なし -->
    <color name="Other_splashBackground">#0080ff</color>
```

**Comment by tateisu** [comment]
- Outlined icon pattern
- Change default button alpha value to 1.0 (full opaque)
- Considering those influences, the color designation of the button was reacquired from the above SS with a dropper.

**Comment by tateisu** [comment]
https://mastodon.juggler.jp/@tateisu/109680333134198731

maybe included at 5.0.8

**Comment by tateisu** [comment]
https://github.com/tateisu/SubwayTooter/releases/tag/v5.0.8
